src/components/Input.tsx

```diff
@@ -1,17 +1,18 @@
+import React from 'react';
+import { Field, ErrorMessage } from 'formik';
+
 const Input = ({
   type,
   name,
   label,
   placeholder,
   helpText,
-  disabled,
 }: {
   type: string;
   name: string;
   label: string;
   placeholder?: string;
   helpText?: string;
-  disabled?: boolean;
```

Comment:
disabled not needed?

Reply:
Yes, for now, we don't need this. If we need this in future then will add it.